package-lock.json

```diff
@@ -61786,22 +61514,6 @@
         "strip-indent": "^3.0.0"
       }
     },
-    "redis": {
-      "version": "4.7.1",
-      "resolved": "https://registry.npmjs.org/redis/-/redis-4.7.1.tgz",
-      "integrity": "sha512-S1bJDnqLftzHXHP8JsT5II/CtHWQrASX5K96REjWjlmWKrviSOLWmM7QnRLstAWsu1VBBV1ffV6DzCvxNP0UJQ==",
-      "dev": true,
-      "optional": true,
-      "peer": true,
-      "requires": {
-        "@redis/bloom": "1.2.0",
-        "@redis/client": "1.6.1",
-        "@redis/graph": "1.1.1",
-        "@redis/json": "1.0.7",
-        "@redis/search": "1.2.0",
-        "@redis/time-series": "1.1.0"
-      }
-    },
```

Comment:
whatever I did manually last time, did not remove this. Maybe it's my `npm@10.8.2` vs the workflow's `npm@11` that's doing something different (this looks like a peer dep pulled in from `typeorm`)

Reply:
looks like `npm@10.9.2` (used in workflow) does something different to what `npm@10.8.2` does locally.